src/app/routes/_app.issues._index.tsx

```diff
@@ -291,7 +304,7 @@ function IssuesTable({ issues }: HTMLAttributes<HTMLTableElement> & { issues: Se
                 );
             },
             cell: ({ row }) => {
-                return <span>{formatDate(new Date(row.getValue('date')))}</span>;
+                return <span>{formatDate(new Date(row.getValue('activity')))}</span>;
```

Comment:
```suggestion
                return <span className='pl-4'>{formatDate(new Date(row.getValue('activity')))}</span>;
```

The alignment is better like this, but when refreshing it _jumps_ to the correct position.

Reply:
Yup, I was checking it out and I ended up with the same code 😅